### PR TITLE
chore: add ping container engine method

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
add ping container engine method
### What issues does this PR fix or reference?

Allow to send the ping REST command to the container engine

### CODE REVIEW

File: packages/main/src/plugin/container-registry.ts [hunk 1]
```diff
@@ -774,6 +774,21 @@ export class ContainerProviderRegistry {
...
+  async pingContainerEngine(providerContainerConnectionInfo: ProviderContainerConnectionInfo): Promise<unknown> {
+    let telemetryOptions = {};
+    try {
+      return this.getMatchingEngineFromConnection(providerContainerConnectionInfo).ping();
+    } catch (error) {
+      telemetryOptions = { error: error };
+      throw error;
+    } finally {
+      this.telemetryService
+
```
Comment (jeffmaury) [inline packages/main/src/plugin/container-registry.ts]: Maybe remove this line

**Review by jeffmaury — APPROVED** [review]
LGTM

File: packages/main/src/plugin/container-registry.ts (hunk 1 above)
Comment (benoitf) [reply packages/main/src/plugin/container-registry.ts:786]: @jeffmaury you don't want to track people checking with ping ?

Comment (benoitf) [reply packages/main/src/plugin/container-registry.ts]: fixed

**Pull request merged**